lib/rucio/tests/test_preparer.py

```diff
@@ -304,26 +303,24 @@ def setup(rse):
 
 
 def test_sort_requests_minimum_distance():
-    request_tuples = [[]] * 3
-    for i in range(3):
-        request_tuples[i] = [0] * 21
-        request_tuples[i][request_id_col] = i
-        request_tuples[i][distance_col] = 3 - i
-
-    result = sort_requests_minimum_distance(request_tuples)
-    assert next(result)[request_id_col] == 2
-    assert next(result)[request_id_col] == 1
-    assert next(result)[request_id_col] == 0
+    request_dicts = [{}, {}, {}]
+    for i in range(len(request_dicts)):
+        request_dicts[i]['request_id'] = i
```

Comment:
i neither understand why this was multiplied with 21, nor why it was removed? :-D

Reply:
The multiplication to build that long tuple was just to return the same format as the refactored `__list_transfer_requests_and_source_replicas` function